### DISCUSSION TITLE
Getting "Cannot get property 'android' on null object"

### DISCUSSION THREAD

**Discussion by markdavies** [invertase/react-native-firebase#5020] [Q&A]
Hi there,

I'm on a Mac trying to get an Android build running on an emulator, but am running into this error when I run `yarn react-native run-android`.

```
A problem occurred evaluating project ':react-native-firebase_analytics'.
> Cannot get property 'android' on null object

* Where:
Build file '<project root>/node_modules/@react-native-firebase/analytics/android/build.gradle' line: 33

```
I have the following packages installed:

```
"@react-native-firebase/analytics": "^11.0.0",
"@react-native-firebase/app": "^11.0.0",
"@react-native-firebase/crashlytics": "^11.0.0",
"@react-native-firebase/messaging": "^11.0.0",
```

I feel like I'm doing something fairly basic incorrectly....

Cheers,
Mark

**Comment by mikehardy** [comment]
Something very fundamental is failing yes

https://github.com/invertase/react-native-firebase/blob/e7d7833f3dacec5e2c80cb87ece08fa06370e3d9/packages/analytics/android/build.gradle#L23-L33

In no case should analytics not be able find the app package and derive the correct SDK versions.

I suspect some old versions of something. This example-project-generator should work, and can be used as a comparison tool to find the badness so you can evict it https://github.com/mikehardy/rnfbdemo/blob/master/make-demo.sh
